hadoop-tools/hadoop-azure/src/main/java/org/apache/hadoop/fs/azurebfs/services/AbfsClient.java

```diff
@@ -194,6 +194,18 @@ private URL changePrefixFromBlobtoDfs(URL url) throws InvalidUriException {
     return url;
   }
 
+  private URL changePrefixFromDfsToBlob(URL url) throws InvalidUriException {
+    if (url.toString().contains(WASB_DNS_PREFIX)) {
+      return url;
+    }
+    try {
+      url = new URL(url.toString().replace(ABFS_DNS_PREFIX, WASB_DNS_PREFIX));
```

Comment:
Same for ABFS_DNS_PREFIX

Reply:
https://github.com/saxenapranav/hadoop/blob/0e76fa384c8091e7ca593b37b4674b41e1c201d1/hadoop-tools/hadoop-azure/src/main/java/org/apache/hadoop/fs/azurebfs/constants/FileSystemUriSchemes.java#L32

its `.dfs.`